soc/arm/renesas_smartbond/da1469x/soc.c

```diff
@@ -25,10 +31,41 @@ static int renesas_da14699_init(void)
 	/* Reset clock dividers to 0 */
 	CRG_TOP->CLK_AMBA_REG &= ~(CRG_TOP_CLK_AMBA_REG_HCLK_DIV_Msk |
 				   CRG_TOP_CLK_AMBA_REG_PCLK_DIV_Msk);
-	/* Enable all power domains except for radio */
-	CRG_TOP->PMU_CTRL_REG = 0x02;
+	
+	CRG_TOP->PMU_CTRL_REG |= (CRG_TOP_PMU_CTRL_REG_TIM_SLEEP_Msk      |
+				              CRG_TOP_PMU_CTRL_REG_PERIPH_SLEEP_Msk   |
+				              CRG_TOP_PMU_CTRL_REG_COM_SLEEP_Msk      |
+				              CRG_TOP_PMU_CTRL_REG_RADIO_SLEEP_Msk);
+	/* PDC should take care of PD_SYS */
+	CRG_TOP->PMU_CTRL_REG &= ~CRG_TOP_PMU_CTRL_REG_SYS_SLEEP_Msk;
+
+	/* 
+	    Due to a crosstalk issue any power rail can potentially
+		issue a fake event. This is typically observed upon power sources 
+		switching, that is DCDC <--> LDOs <--> Retention LDOs
+     */
+	CRG_TOP->BOD_CTRL_REG &= ~(CRG_TOP_BOD_CTRL_REG_BOD_V14_EN_Msk  |
+	                           CRG_TOP_BOD_CTRL_REG_BOD_V18F_EN_Msk |
+							   CRG_TOP_BOD_CTRL_REG_BOD_VDD_EN_Msk  |
+							   CRG_TOP_BOD_CTRL_REG_BOD_V18P_EN_Msk |
+							   CRG_TOP_BOD_CTRL_REG_BOD_V18_EN_Msk  |
+							   CRG_TOP_BOD_CTRL_REG_BOD_V30_EN_Msk  |
+							   CRG_TOP_BOD_CTRL_REG_BOD_VBAT_EN_Msk);
```

Comment:
indentation in added lines is broken

Reply:
To be honest, I do not know how to fix this and why indentation has been broken.

Reply:
I think it should be OK now.